### ISSUE TITLE
Buttons (e.g., Edit, Save) have no focus indicator in Chrome

### ISSUE THREAD

**Issue by AmeliaBR** [mavoweb/mavo#267]
When playing with the demos in Chrome, I was completely lost because the keyboard focus wasn't showing up for the buttons in the Mavo tool bar (Edit, Save, Clear).

The problem is the `all:unset` declaration in the button styles, which are assigned with class selectors that override the `:focus` selector in the browser style sheets.   So it unsets the normal focus outline.

Firefox still shows an outline, but that seems to be because they have an over-ride that adds the default focus ring back in, if the normal CSS rules result in no focus styles at all--probably to address cases of devs _intentionally_ removing the focus ring. (And Edge just ignores the `all: unset` completely).

The CSS needs to be changed to either use more specific resets (instead of `all`), or to add custom `:focus` indicator styles.
